src/app/posting/[slug]/page.tsx

```diff
@@ -15,12 +15,43 @@ interface Props {
 	params: Promise<Params>;
 }
 
+function requireToken() {
+	const env = {
```

Comment:
```suggestion
        const GitEnv = {
```
GitHub 관련 환경 변수이니 GitEnv라고 네이밍하는 게 조금 더 직관적일 것 같은데, 어떻게 생각하시나요~?

( 이 부분은 제가 Commit suggestion누르면 어떻게 되는지 궁금해서 한번 눌러봤는데 이미 파일에 반영돼버렸네요....😥😥 )
그래도 의견 남겨주세요..!!

Reply:
( 이 부분은 제가 Commit suggestion누르면 어떻게 되는지 궁금해서 한번 눌러봤는데 이미 파일에 반영돼버렸네요....😥😥 )
그래도 의견 남겨주세요..!!

Reply:
좋아요! 변경해주신 환경변수명이 더 명확한 것 같습니다